game-core/src/main/java/games/strategy/triplea/ai/pro/ProCombatMoveAi.java

```diff
@@ -48,7 +48,7 @@
 /**
  * Pro combat move AI.
  */
-class ProCombatMoveAi {
+public class ProCombatMoveAi {
```

Comment:
This change is just there so my IntelliJ doesn't complain that this class is referenced in a javadoc outside of this package